Wikipedia/Code/ArticlePlaceView.swift

```diff
@@ -3,14 +3,20 @@ import MapKit
 import WMF
 
 class ArticlePlaceView: MKAnnotationView {
-    private let imageView: UIImageView
-    private let selectedImageView: UIImageView
+    private let imageView: UIView
```

Comment:
Originally it was simple enough to not need IB, now, given all of the overlapping views and hierarchies, it's easier to not use IB